components/src/CartLineItem/cart.lineitem.tsx

```diff
@@ -136,90 +132,49 @@ class CartLineItem extends React.Component<CartLineItemProps, CartLineItemState>
     this.setState({ quantity: newItemQuantity });
   }
 
-  handleConfiguratorAddToCartBtnClicked() {
+  async handleConfiguratorAddToCartBtnClicked() {
     const {
       item, handleQuantityChange, handleErrorMessage, itemQuantity, onConfiguratorAddToCart,
     } = this.props;
     handleQuantityChange();
-    login().then(() => {
-      const addToCartLink = item._addtocartform[0].links.find(link => link.rel === 'addtodefaultcartaction');
-      cortexFetch(addToCartLink.uri,
-        {
-          method: 'post',
-          headers: {
-            'Content-Type': 'application/json',
-            Authorization: localStorage.getItem(`${Config.cortexApi.scope}_oAuthToken`),
-          },
-          body: JSON.stringify({
-            quantity: itemQuantity || 1,
-          }),
-        })
-        .then((res) => {
-          if (res.status === 200 || res.status === 201) {
-            onConfiguratorAddToCart();
-          } else {
-            res.json().then((json) => {
-              handleErrorMessage(json);
-            });
-          }
-        })
-        .catch((error) => {
-          // eslint-disable-next-line no-console
-          console.error(error.message);
-        });
-    });
+    const quantity = itemQuantity || 1;
```

Comment:
Quantity is a number here. It should be consistent and be either a number all the time or a string all the time.